workbench/tests/invest/flaskapp.test.js

```diff
@@ -214,6 +214,8 @@ describe('Test building model UIs and forum links', () => {
         modelID={modelID}
         argsSpec={argsSpec.args}
         userguide={argsSpec.userguide}
+        isCoreModel={true}
+        uiSpec={argsSpec.ui_spec}

```

Comment:
I think as of the latest `feature/plugins`, `uiSpec` is no longer needed as a prop and `inputFieldOrder` is used instead.